### ISSUE TITLE
Radarr not importing movie from Sabnzbd

### ISSUE THREAD

**Issue by TheCreatorzOne** [Radarr/Radarr#4236]
**Describe the bug**
Radarr seems to not pick up some movies from sabnzbd. Most movies go fine but some just outright disappear from the queue after being added and are not getting picked up once downloaded. Radarr seems to see the movie but does not parse it. I've attached a debug log.

**To Reproduce**
1. Tried with a specific release for Nymphomaniac Vol 1

**Expected behavior**
Radarr should be able to pick up and process all movies added by radarr

**Desktop (please complete the following information):**
 - OS: Ubuntu (Dockerized/Linuxserver.io image for aphrodite)
 - Mono Version: [e.g. Mono 5.8] Not using Mono
 - Browser and Version [e.g. chrome, safari] N/A
 - Version [e.g. 22] 3.0.0.2467

**Debug Logs**
Turn on debug logs under Settings -> General and wait for the bug to occur again. **Upload the full log file here (or another site and link it). Issues will be closed, if they do not include this!**

[radarr.debug.txt](https://github.com/Radarr/Radarr/files/4349866/radarr.debug.txt)

The bottom line is indicating some issue where it does not find the movie, even though further up it finds it and maps it correctly
